nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -133,6 +133,21 @@ The `ipam.Role`, `dcim.RackRole`, and `dcim.DeviceRole` models have been removed
 |----------------|----------------------|--------------------------------------|
 | `/api/status/` | `rq-workers-running` | Removed as RQ is no longer supported |
 
+### Removed Versioned Endpoints and Serializers
+
+Nautobot 2.0 removes support for 1.X versioned REST APIs and their Serializers. Requesting [older API versions](../rest-api/overview.md#versioning) will result in a `400 Bad Request` error.
+
+**Endpoints Affected:**
+
+- `/api/dcim/interfaces`
+- `/api/ipam/ip-addresses`
+- `/api/extras/custom-fields`
+- `/api/extras/jobs`
+- `/api/extras/tags`
+- `/api/virtualization/interfaces`
+
+Please ensure you are using the latest representations of request/response representations as seen in the API docs or Swagger.
+
```

Comment:
@glennmatthews thoughts?

Reply:
Hmm. I worry that people are going to misread this as saying we've removed those API endpoints altogether.

Requesting 1.x API versions of *any* REST API endpoint will fail after this changeset, so I'm not sure what's gained by calling out these ones specifically?

Reply:
> Hmm. I worry that people are going to misread this as saying we've removed those API endpoints altogether.
> 
> Requesting 1.x API versions of _any_ REST API endpoint will fail after this changeset, so I'm not sure what's gained by calling out these ones specifically?

My thought was to draw attention to the endpoints vs the version. I suspect integrators know more which endpoints they hit than the versions of which endpoints they hit.

Maybe rephrasing the list heading to something like `Endpoints with Legacy Versions`?